piperider_cli/assertion_engine/assertion.py

```diff
@@ -211,10 +211,10 @@ def check_assertions_syntax(assertion_search_path=PIPERIDER_ASSERTION_SEARCH_PAT
         """
         return load_yaml_configs(assertion_search_path)
 
-    def load_assertions(self):
+    def load_assertions(self, assertions_content={}):
```

Comment:
The `assertions_content={}` in function argument will hit the #1 Common Python Mistake
`Misusing expressions as defaults for function arguments`
ref: https://www.toptal.com/python/top-10-mistakes-that-python-programmers-make